app/assets/javascripts/mobile/mobile_comments.js

```diff
@@ -11,6 +11,8 @@
     initialize: function() {
       var self = this;
 
+      new Diaspora.MarkdownEditor(".comment_box");
```

Comment:
Then it can be done this way:
```
new Diaspora.MarkdownEditor(".comment_box").render();
```

My main point is to split object creation and its actual usage.

I don't think it major (and moreover out of the PR scope), but what do others think?